assets/src/edit-story/elements/text/display.js

```diff
@@ -103,10 +131,37 @@ function TextDisplay({
       : '';
   });
 
+  if (backgroundType === 'highlight') {
+    return (
+      <>
+        <Original>
+          <HighlightElement
+            ref={ref}
+            dangerouslySetInnerHTML={{
+              __html: draftMarkupToContent(content, bold),
+            }}
+            {...props}
+          />
+        </Original>
+        <Clone {...props}>
+          <HighlightElement
+            ref={ref}
+            dangerouslySetInnerHTML={{
+              __html: draftMarkupToContent(content, bold),
+            }}
+            {...props}
+          />
+        </Clone>
+      </>
```

Comment:
I'm not sure I understand this. Why do we need a clone?

Reply:
Actually, removing that and improving the example, I am being able to remove that clone node.